cmds/core/yes/yes_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestMain(m *testing.M) {
+	testutil.Run(m, main)
+}
+
+func TestYes(t *testing.T) {
+	cmd := testutil.Command(t)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		t.Logf("Could not return a pipe that will be connected to the command's standard output, received err: %v", err)
```

Comment:
If you can't get a pipe. you can't continue. You should use t.Fatalf here.

Reply:
Yup, that is a good point. Fatal is stop the execution at that point. Thank you for pointing it out to me :)!